eyeball-im/src/reusable_box.rs

```diff
@@ -0,0 +1,161 @@
+// Copy-pasted from https://docs.rs/tokio-util/latest/src/tokio_util/sync/reusable_box.rs.html
```

Comment:
Should we add the license, or create an `ATTRIBUTION` file?

Reply:
Is that some sort of standard? I didn't find anything immediately.

I felt when writing this that the comment at the top should be sufficient attribution for copying a small file out of a permissively-licensed project. I suppose it wouldn't hurt to note the original license. I don't think adding the license anywhere is necessary though, since MIT allows redistributing stuff under a different license IIRC (I am not a lawyer, of course).

Also, I plan to get rid of this module again in the future anyways, once https://github.com/tokio-rs/tokio/pull/6908 is merged and another release is made it should be possible to write a self-referential future by hand that can be stored in the `VectorSubscriberStream` without going through `dyn Future`. It's a bit tricky though, and I haven't had any focus time for that sort of stuff lately. Let me know if you're interested in helping out 😉